variables.tf

```
@@ -112,6 +117,25 @@ variable "tags" {
   default     = {}
 }
 
+# API Association & Domain Name
+variable "domain_name" {
+  description = "The domain name that AppSync gets associated with."
+  type        = string
+  default     = ""
+}
+
+variable "domain_name_description" {
+  description = "A description of the Domain Name."
+  type        = string
+  default     = ""
```

```suggestion
  default     = null
```
Since it is optional